app/build.gradle

```diff
@@ -129,8 +131,9 @@ dependencies {
 
     implementation fileTree(dir: 'libs', include: ['*.jar'])
 
-    //noinspection GradleDependency - 1.1 has a bug which causes a crash
-    implementation "com.android.installreferrer:installreferrer:$referrerLibrary"
+    implementation("com.android.installreferrer:installreferrer:$referrerLibrary") {
+        because "noinspection GradleDependency - 1.1 has a bug which causes a crash"
```

Comment:
💥

Reply:
1.1 **had** a crash. They fixed it, and released it in another version of 1.1 🤦‍♂️